src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -4938,6 +4938,10 @@ public function remove_class( $class_name ): bool {
 	 *
 	 * @since 6.6.0 Subclassed for the HTML Processor.
 	 *
+	 * @todo When reconstructing active formatting elements with attributes, find a way
+	 *       to indicate if the virtually-reconstructed formatting elements contain the
+	 *       wanted class name.
+	 *
 	 * @param string $wanted_class Look for this CSS class name, ASCII case-insensitive.
 	 * @return bool|null Whether the matched tag contains the given class name, or null if not matched.
 	 */
```

Comment:
@sirreal I've reverted this change so we can consider it separately. I know it will be important during active format reconstruction, but I think that `null` is a kind of partially-implemented escape hatch that communicates that this isn't supported rather than indicating that a class definitively doesn't exist on the tag.